components/popover_list_members/popover_list_members.jsx

```diff
@@ -60,24 +54,22 @@ export default class PopoverListMembers extends React.Component {
     }
 
     componentWillReceiveProps(nextProps) {
-        if (!Utils.areObjectsEqual(this.props.members, nextProps.members)) {
-            const sortedMembers = this.sortMembers(nextProps.members);
-            const teamMembers = UserStore.getProfilesUsernameMap();
+        if (!Utils.areObjectsEqual(this.props.users, nextProps.users) || !Utils.areObjectsEqual(this.props.statuses, nextProps.statuses)) {
```

Comment:
Is the cost of invoking `this.sortUsers` strictly worse than comparing `this.props.users` and `this.props.statuses` to the incoming properties? Should we consider just sorting on each render, or memoizing this in a selector?

Reply:
Invoking `this.sortUsers` also causes `this.state.sortedUsers` to change, so it rerenders this component and all its children. The method is not good. I'll see if I can clean it up a bit